oeps/processes/oep-0054-core-contributors.rst

```diff
@@ -216,10 +220,10 @@ The process for nomination is as follows:
 #. At the end of the comment period, a candidate is approved if there are at
    least 5 (five) affirmative "yes" votes, and zero "no" votes.
 
-   #. For candidates coming from an organization, at least one affirmative vote
-      must come from a CC outside of the candidate's org. This is to ensure that
-      core contributors are evaluated on their ability to communicate with the
-      community, outside of their org.
+   #. For candidates coming from an organization, either the nomination or at
```

Comment:
This doc uses the word "sponsor" to refer to the person who puts forth the nomination. Maybe...

...either the sponsor, or at least one of the affirmative votes, must come from...

Reply:
Ah, yeah, definitely don't want to unnecessarily introduce new terminology. Will fix.